addons/hr_expense/tests/test_expenses.py

```diff
@@ -686,6 +686,31 @@ def test_attachments_in_move_from_own_expense(self):
             }
         ])
 
+    def test_no_change_in_price_unit_with_expense_no_extract(self):
+        """
+        Test that the price unit does not change when the quantity changes after uploading an attachment
+        Some of the logic of hr_expense_extract was applied here.
```

Comment:
Then why not put the test in `hr_expense_extract` :thinking: ?

Reply:
I was hesitating. Since we do not change the logic in `hr.expense.extract` and we just reestablish what a standard flow should be in `hr.expense`.

```quote
Note:
This will not solve the flow in which a user upload an attachment, has credit and has the option "digitize automatically". Everytime the user will upload an attachment the OCR will be triggered and with enough credit the document will be processed (even if it is a silly image)
```